BrainPortal/app/models/cluster_task.rb

```diff
@@ -2176,6 +2177,21 @@ def update_size_of_cluster_workdir
     return nil
   end
 
+  # This will increase the "task_setup" counts of the
+  # DataUsage records for any and all userfiles left in
+  # the :interface_userfile_ids array of the task's params.
+  def track_usage_of_input_files_after_setup #:nodoc:
```

Comment:
I think it's possible for a user to select X files and to use only a part of these X files. In this case, should we increase the count for all the files?